code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -196,6 +196,10 @@ emp_act
 	return 0
 
 /mob/living/carbon/human/proc/check_reflect(def_zone) //Reflection checks for anything in your l_hand, r_hand, or wear_suit based on the reflection chance var of the object
+	if(head && istype(head, /obj/item/))
+		var/obj/item/I = head
+		if(I.IsReflect(def_zone) == 1)
+			return 1
```

Comment:
Head is already defined as `obj/item` in `carbon_defines.dm`, so you don't need to do this extra checking here

```suggestion
	if(head.IsReflect(def_zone) == 1)
		return 1
```

Reply:
ack. Should be good now, thank you.